app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\CreateUserRequest;
+use App\Models\User;
+use Illuminate\Pagination\Paginator;
+class UserController extends Controller
+{
+    public function index() {
+        $user = User::showUser();
```

Comment:
Function này dùng để list toàn bộ user  em đặt showUser() không hợp lí.
biến $user lên đặt số nhiều thì hợp lí hơn nhé ví dụ:
`$users = User::listUser();`